docs/plugins/02-default-plugins.md

```diff
@@ -106,6 +106,18 @@ View the documentation with `:help luasnip`
 
 LuaSnip uses friendly-snippets as it's snippet library.
 
+```lua
+-- Beside defining your own snippets you can also load snippets from "vscode-like" packages
+require("luasnip/loaders/from_vscode").load({ paths = { "./my-snippets" } })
```

Comment:
I'm assuming this is unrelated to the lines after. Doesn't LuaSnip also looks for a `snippets` folder on your `runtimepath`? meaning, you can create `~/.config/lvim/snippets`.

Reply:
Not 100% sure, but I don't think it does. You can store snippets wherever you want, so it can be shared folder with vscode for example. Just point line above to folder with package.json file, which will take care of the rest. Example:
https://github.com/dsznajder/vscode-es7-javascript-react-snippets/blob/2a6a1ffac598d7f5b4097d06c4190c5bcced99d9/package.json#L50

I'd assume this is what most people are looking for, or at least I was. 😅

Reply:
@abzcoding, what's your take on this, since you might have some experience with it.

Reply:
i'm also doing this
```lua
require("luasnip/loaders/from_vscode").load { paths = { "~/.config/lvim/snippets" } }
```
and i have a `~/.config/lvim/snippets/package.json` which contains
```json
{
  "name": "nvim-snippets",
  "author": "abzcoding",
  "engines": {
    "vscode": "^1.11.0"
  },
  "contributes": {
    "snippets": [
      {
        "language": "lua",
        "path": "./lua.json"
      },
      {
        "language": [
          "typescript",
          "typescriptreact"
        ],
        "path": "./typescript.json"
      },
      {
        "language": "typescriptreact",
        "path": "./react-ts.json"
      }
    ]
  }
}
```
so i'm not sure my approach is the best one 😅

Reply:
My approach is otherwise the same, but I keep my snippets separate from my dotfiles.

Reply:
Thanks for this PR. Sadly this does not quite work for me. I added 

```lua
require("luasnip/loaders/from_vscode").load { paths = { "~/.config/lvim/mysnippets" } }
```
somewhere in my `config.lua` and created `"~/.config/lvim/mysnippets/package.json"` along with a snippet file. 
* However the snippets don't work and are not found by `:LuaSnipListAvailable`
* Also if I do
```lua
require("luasnip/loaders/from_vscode").load { paths = { "does not exist" } }
```
I don't get any error message. Am I missing something? Any idea how to debug this?

Reply:
this is what i get
<img width="1912" alt="Screen Shot 2022-01-03 at 4 48 11 PM" src="https://user-images.githubusercontent.com/10992695/147934944-396151a4-01cb-43e2-87b9-62ce9d1b7e5d.png">

this is what i used 
```lua
  require("luasnip/loaders/from_vscode").load { paths = { "~/.config/lvim/snippets" } }
```
and here are my snippets -> https://github.com/abzcoding/lvim/tree/main/snippets

Reply:
> 

I'ld try with existing snippet package. Clone https://github.com/dsznajder/vscode-es7-javascript-react-snippets for example and if that works you know problem is in your own package.json or other snippet files and you can mimic that package.

Reply:
Thanks, @sambergo @abzcoding for the quick reply. The react snippets work, so it must be an issue with my snippets. Sorry for the noise (if somebody knows how to get an error message or other diagnostics on the snippets that would be awesome).

Reply:
> I'm assuming this is unrelated to the lines after. Doesn't LuaSnip also looks for a `snippets` folder on your `runtimepath`? meaning, you can create `~/.config/lvim/snippets`.

Yes it does. So if you have `~/.config/lvim/snippets/my-snippets/package.json` you can do both:
`require("luasnip/loaders/from_vscode").load { paths = { "~/.config/lvim/snippets/my-snippets" } }`
or
`require("luasnip/loaders/from_vscode").load { paths = { "my-snippets" } }`